x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -103,12 +103,6 @@ export const allowedExperimentalValues = Object.freeze({
    * Enables experimental Entity Analytics HTTP endpoints
    */
   riskScoringRoutesEnabled: true,
-  /*
-   *
-   * Enables Discover embedded within timeline
-   *
-   * */
-  discoverInTimeline: true,
```

Comment:
Removed in favor of the config created here: https://github.com/elastic/kibana/pull/167856